### PR TITLE
Feature/lmx 878

### PR DESCRIPTION
**Pull request by degaray**

### CODE REVIEW

File: src/CostaLine/Type/Seat.php [hunk 1]
```diff
@@ -89,4 +90,20 @@ public function getYPos()
     {
         return $this->yPos;
     }
+
+    /**
+     * @return mixed
+     */
+    public function getHasTV()
```
Comment (pollin14) [inline src/CostaLine/Type/Seat.php]: The PHPDoc is wrong. Return a boolean. Also the setter.

File: src/CostaLine/Type/SeatMap.php [hunk 2]
```diff
@@ -11,7 +11,7 @@ class SeatMap
     public function addSeat($seat)
     {
```
Comment (pollin14) [inline src/CostaLine/Type/SeatMap.php]: An extra white space.

**Comment by pollin14** [comment]
Code review. OK.

**Comment by Ajaxman** [comment]
Code review is ok, but this branch will be merged in RC 1.41.0

**Comment by pollin14** [comment]
jaja

**Pull request closed without merging**